sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/FileScan.scala

```diff
@@ -42,6 +44,21 @@ abstract class FileScan(
     false
   }
 
+  override def description(): String = {
+    def seqToString(seq: Seq[Any]) = seq.mkString("[", ", ", "]")
+    val locationDesc = fileIndex.getClass.getSimpleName + seqToString(fileIndex.rootPaths)
```

Comment:
`seqToString(fileIndex.rootPaths)` -> `fileIndex.rootPaths.mkString("[", ", ", "]")`. And, this may need new line.